config/nginx-proxy-common.yaml

```diff
@@ -0,0 +1,11 @@
+# Common values for all nginx-proxies
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
```

Comment:
Good for starting, but it seems a bit low for the CPU. Take care of checking the datadog metrics (CPU load especially)